lib/deaf_grandma.rb

```diff
@@ -19,9 +19,17 @@ def run!
 
 
   def speak(input)
-
-    #Implement your code here <<<<<<<<<
-
+    upper = /[A-Z]+/
+    lower = /[A-Z]?[a-z]+/
+    bye = /\A(BYE ?){3}\z/
+    response = ''
```

Comment:
I don't think we need this local variable. Below, you conditionally assign different values to it, but you don't do anything else with `response`. I'm pretty sure this method will behave the same if you don't have this local variable and just put those three strings in each part of the `if`/`elsif`/`elsif`.